modules/publish/templates/_specialcontributions.inc.php

```diff
@@ -0,0 +1,100 @@
+        <div class="header"><?php echo __('Special:Contributions to %namespace namespace by %username',
+                                          array('%namespace' => ($projectnamespace ? $projectnamespace : __('global')),
+                                                '%username' => $user)); ?>
+
+        </div>
+        <p>
+            <?php echo __('Below is a listing of all contributions made by this user to %namespace namespace. Contributions are listed only for articles you are allowed to access.',
+                          array('%namespace' => ($projectnamespace ? $projectnamespace : 'global'))); ?>
+        </p>
+    <?php endif ?>
+
+    <?php if (!empty($contributions)): ?>
+        <div class="full_width_table outter">
+            <table class="full_width_table inner">
+                <thead>
+                    <tr>
+                        <th><?php echo __('Date'); ?></th>
+                        <th><?php echo __('Details'); ?></th>
+                        <th><?php echo __('Article'); ?></th>
+                        <?php if ($username === null): ?>
+                            <th><?php echo __('Author'); ?></th>
+                        <?php endif ?>
+                        <th><?php echo __('Reason'); ?></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <?php for ($i = $page_start_at; $i <= $page_end_at; $i++): ?>
+                        <tr>
+                            <td><?php echo link_tag($contributions[$i]['revision_url'], tbg_formatTime($contributions[$i]['date'], 1)); ?></td>
+                            <td>(
+                                <?php echo ($contributions[$i]['diff_url']) ? link_tag($contributions[$i]['diff_url'], 'diff') : 'diff'; ?>
+                                |
+                                <?php echo link_tag($contributions[$i]['history_url'], 'hist'); ?>
+                                )</td>
+                            <td><?php echo link_tag($contributions[$i]['article_url'], $contributions[$i]['article_name']); ?></td>
+                            <?php if ($username === null): ?>
+                                <?php if ($contributions[$i]['author'] !== null): ?>
+                                    <td><?php echo link_tag($contributions[$i]['author_contributions_url'], $contributions[$i]['author']); ?></td>
+                                <?php else: ?>
+                                    <td><em><?php echo __('no author'); ?></em></td>
+                                <?php endif?>
+                            <?php endif ?>
+                            <td><?php echo $contributions[$i]['reason']; ?></td>
+                        </tr>
+                    <?php endfor ?>
+                </tbody>
+            </table>
+        </div>
+        <div>
+            View (
+            <?php echo ($page == 1 ? __('newest') : link_tag($navigation_urls['newest'], __('newest'))); ?> |
```

Comment:
Ok, will do. Didn't know about that one, plus saw most of the old code used the echo :)